cfpb/consumerfinance.gov
# Cleanup filter lists

Removing old/dead/unused code now that we've migrated our filterable lists to elasticsearch.

---


## Removals

- Code related to using django's orm for filterable lists
- Flag for enabling elasticsearch filterable lists
- Unused filter_sibling option


## How to test this PR

1. Repopulate ES Indices
2. Search for pages and confirm results match expectations


## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
